TestLib/Exception/GREYDefaultFailureHandler.m

```diff
@@ -81,23 +94,46 @@ - (void)handleException:(GREYFrameworkException *)exception details:(NSString *)
       [GREYFailureScreenshotSaver saveFailureScreenshotsInDictionary:appScreenshots
                                                          toDirectory:screenshotDir];
   NSAssert(screenshotPaths, @"Screenshots must be present");
+  return screenshotPaths;
+}
 
-  NSString *appUIHierarchy = [exception.userInfo valueForKey:kErrorDetailAppUIHierarchyKey];
-  // For calls from GREYAsserts in the test side, the hierarchy must be populated here.
-  if (!appUIHierarchy) {
-    appUIHierarchy = [GREYElementHierarchy hierarchyString];
+- (NSString *)consoleLogForException:(GREYFrameworkException *)exception
```

Comment:
Please add comments for this and other functions created.

Reply:
done